src/context/Create.tsx

```diff
@@ -132,6 +133,42 @@ const CreateProvider = (props: { children: any }) => {
     const [boltzFee, setBoltzFee] = createSignal(0);
     const [minerFee, setMinerFee] = createSignal(0);
 
+    // embed
+    const invoiceParam = detectUrlParam("invoice");
+    if (invoiceParam && invoiceParam !== "") {
+        setInvoice(invoiceParam);
+    }
+    const addressParam = detectUrlParam("address");
+    if (addressParam && addressParam !== "") {
+        setOnchainAddress(addressParam);
+    }
+    // only set assets if both are set and not equal
+    const assetSendParam = detectUrlParam("assetSend");
+    const assetReceiveParam = detectUrlParam("assetReceive");
+    if (
+        assetSendParam &&
+        assetSendParam !== "" &&
+        assetReceiveParam &&
+        assetReceiveParam !== "" &&
+        [BTC, LN, LBTC, RBTC].includes(assetSendParam) &&
+        [BTC, LN, LBTC, RBTC].includes(assetReceiveParam) &&
+        assetSendParam !== assetReceiveParam
```

Comment:
Why not just check if that pair is valid based on the API responses we fetch on load?